motr/m0crate/parser.c

```diff
@@ -442,6 +433,32 @@ int copy_value(struct workload *load, int max_workload, int *index,
 			else
 				parser_emit_error("Unkown key ordering: '%s'", value);
 			break;
+		case KEY_SIZE:
+			w = &load[*index];
+			ciw = workload_index(w);
+			if (!strcmp(value, "random"))
+				ciw->key_size = -1;
+			else
+				ciw->key_size = parse_int(value, KEY_SIZE);
+			break;
+		case VALUE_SIZE:
+			w = &load[*index];
+			ciw = workload_index(w);
+			if (!strcmp(value, "random"))
+				ciw->value_size = -1;
+			else
+				ciw->value_size = parse_int(value, VALUE_SIZE);
+			break;
+		case MAX_KEY_SIZE:
+			w = &load[*index];
+			ciw = workload_index(w);
+			ciw->max_key_size = parse_int(value, MAX_KEY_SIZE);
+			break;
+		case MAX_VALUE_SIZE:
+			w = &load[*index];
+			ciw = workload_index(w);
+			ciw->max_value_size = parse_int(value, MAX_VALUE_SIZE);
+			break;
```

Comment:
Please make sure all parameters have a default value even if these parameters are not specified in a .yaml file.

In my testing, if some of these parameters are not specified in .yaml, the m0crate will panic and sometimes, panic the whole OS.

Reply:
Fixed.